projects/2DoBox-Pivot.md

```diff
@@ -0,0 +1,160 @@
+# 2DoBox Pivot
+
+Do you remember that project Ideabox?
+
+We thought they were pretty great, but, I think it's time to pivot those projects over to something we like to call 2DoBox.
+
+You will be inheriting someone's previous IdeaBox and pivoting it over to 2DoBox. Your projects will utilize webpack and it's built in node server. You will also keep accessibility in mind as we will be grading you based on how accessible your application is. You will also implement selenium webdriver to test the DOM.
+
+# Phase 1
+
+### Adding a new ToDo
+
+On the application’s main page, a user should:
+
+See two text boxes for entering the “Title” and “Task” for a new TODO, and a “Save” button for committing that TODO.
+When a user clicks “Save”:
+
+A new TODO with the provided title and body should appear in the TODO list.
+The text fields should be cleared and ready to accept a new TODO.
+The page should not reload.
+The TODO should be persisted. It should still be present upon reloading the page
+
+
+### Deleting an existing TODO
+
+When viewing the TODO list:
+
+Each TODO in the list should have a link or button to “Delete” (or 𝗫).
+Upon clicking “Delete”, the appropriate TODO should be removed from the list.
+The page should not reload when an idea is deleted.
+The TODO should be removed from localStorage. It should not re-appear on next page load.
```

Comment:
Is there a part where they have to mark a todo 'done'?

Typical functionality that I would expect for a todolist app would be:

User can click to show the idea as 'done'

'done' ideas should be greyed out and/or shown with a strikethrough on the text

On reloading the page, done 'todo' items should either display on the bottom of the list OR only be visible if you toggle 'show completed todos'